GlideAggregate/List the incident priority count under each category/code.js

```diff
@@ -0,0 +1,20 @@
+// Get the count of Incidents by Category and then by Priority.
+
+var incCATGR = new GlideAggregate('incident');
+incCATGR.addAggregate('COUNT', 'category');
+incCATGR.orderBy('category');
+incCATGR.query();
+
+while (incCATGR.next()) {
+	var cat = incCATGR.category;
```

Comment:
You don't use this variable, and additionally should use getters to avoid relying on GlideElement casting